Userland/Libraries/LibGfx/Point.h

```diff
@@ -227,6 +227,13 @@ class Point {
         return AK::hypot<float>(m_x - other.m_x, m_y - other.m_y);
     }
 
+    [[nodiscard]] float distance_from(T const& x, T const& y) const
+    {
+        if (m_x == x && m_y == y)
+            return 0;
+        return AK::hypot<float>(m_x - x, m_y - y);
+    }
```

Comment:
Instead of adding this function, I believe you should be able to do `point.distance_from({ x, y })` and the compiler will effectively generate the same code for you.

Reply:
Hey @GMTA,
thanks for reviewing my pr. My intentsion here was to avoid creating a Point object while plotting the line. I was not aware that such stuff gets optimized anyways...
I'll update this pr with your recommendations.

Thanks